readme.md

```diff
@@ -625,6 +639,97 @@ Defaults[.setUser].first?.name //=> "Hello"
 Defaults[.dictionaryUser]["user"]?.name //=> "Hello"
 ```
 
+### Dynamic value
+
+There might be situations where you want to use `[String: Any]` directly.
+But `Defaults` need its value conforms to `Defaults.Serializable`, so here is a class `Defaults.AnySerializable` to overcome this limitation.
+
+`Defaults.AnySerializable` only available for `Value` which conforms to `Defaults.Serializable`.
+
+#### Primitive type
+
+`Defaults.AnySerializable` conforms to `ExpressibleByStringLiteral`, `ExpressibleByIntegerLiteral`, `ExpressibleByFloatLiteral`, `ExpressibleByBooleanLiteral`, `ExpressibleByNilLiteral`, `ExpressibleByArrayLiteral` and `ExpressibleByDictionaryLiteral`.
+
+So it can assign directly with these primitive types.
```

Comment:
I think we should have a clear warning that the type erasure should only be used when there's no other way to handle it. It has much worse performance.

And don't forget to keep the code documentation comments and readme in sync when making changes.